aries-site/src/pages/components/checkbox.js

```diff
@@ -24,11 +32,132 @@ const CheckBox = () => (
         </SubsectionText>
         <Example
           docs="https://v2.grommet.io/checkbox?theme=hpe#props"
-          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/input/CheckboxExample.js"
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/checkbox/CheckboxSimpleExample.js"
+          figma="https://www.figma.com/file/7Mm1xDBTOtPHqggEVpaD2N/HPE-Checkbox-Component?node-id=1744%3A1"
+          designer="https://designer.grommet.io/home?id=HPE-design-system-taylor-seamans-hpe-com"
+        >
+          <CheckBoxSimpleExample />
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          CheckBox can be used to present an individual option or group of
+          options to the user. When presented in a group, the user can select
+          one or multiple options. Refer to checkbox group for examples.
+        </SubsectionText>
+        <SubsectionText>
+          CheckBox can be used in the form of a toggle, a single option which
+          can indicate something such as on or off. Toggle is prefered to
+          quickly switch between two possible states. Toggles usually provides
+          immediate results in which gives users the ability to control their
+          preference.
+        </SubsectionText>
+        <SubsectionText>
+          CheckBox should not be used if a user should only be allowed to select
+          one option from a list of options. In the case that a user should only
+          be able to select one option, refer to{' '}
+          <Anchor href="/components/radiobutton">RadioButton</Anchor>
```

Comment:
```suggestion
          <Anchor href={nameToPath('RadioButton')}>RadioButton</Anchor>.
```

`nameToPath` will also need to be added to imports